snap/info.go

```diff
@@ -231,6 +232,18 @@ type DownloadInfo struct {
 	Sha3_384 string `json:"sha3-384,omitempty"`
 }
 
+// DeltaInfo contains the information to download a delta
+// from one revision to another.
+type DeltaInfo struct {
+	FromRevision    int    `json:"from-revision,omitempty"`
+	ToRevision      int    `json:"to-revision,omitempty"`
```

Comment:
Okay, that's actually nice and explicit indeed feels better, thanks for explaining.